modules/util.py

```diff
@@ -350,3 +377,26 @@ def mcopy(dev, src, dst):
         log.debug("Failed to mcopy in " + dev);
         return False
     return True
+
+def jsonAttributeExists(json, attribute):
```

Comment:
Why are we using `jq` for reading and writing JSON? https://docs.python.org/3/library/json.html